ci/scenarios/vPool/add_remove_vpool_concurrently/main.py

```diff
@@ -35,32 +35,33 @@ class VPoolTester(CIConstants):
     LOGGER = Logger('scenario-{0}'.format(TEST_NAME))
 
     def __init__(self):
-        self.LOGGER.info("Initializing concurrent vpool testing.")
-        self.valid_storagerouters = []
+        VPoolTester.LOGGER.info("Initializing concurrent vpool testing.")
+        VPoolTester.valid_storagerouters = []
         for storagerouter in StoragerouterHelper.get_storagerouters():
             try:
                 RoleValidation.check_required_roles(VPoolSetup.REQUIRED_VPOOL_ROLES, storagerouter.ip, "LOCAL")
-                self.valid_storagerouters.append(storagerouter)
-                self.LOGGER.info('Added storagerouter {0} to list of eligible storagerouters'.format(storagerouter.ip))
+                VPoolTester.valid_storagerouters.append(storagerouter)
+                VPoolTester.LOGGER.info('Added storagerouter {0} to list of eligible storagerouters'.format(storagerouter.ip))
             except RuntimeError:
                 pass
-        assert len(self.valid_storagerouters) >= 1, 'At least one storagerouter with valid roles required: none found!'
-        self.alba_backends = BackendHelper.get_alba_backends()
-        assert len(self.alba_backends) >= 1, 'At least one backend required, none found!'
-        self.vpools = []
+        assert len(VPoolTester.valid_storagerouters) >= 1, 'At least one storagerouter with valid roles required: none found!'
+        VPoolTester.alba_backends = BackendHelper.get_alba_backends()
+        assert len(VPoolTester.alba_backends) >= 1, 'At least one backend required, none found!'
+        VPoolTester.vpools = []
         super(VPoolTester, self).__init__()
 
-    def add_vpools(self):
+    @classmethod
+    def add_vpools(cls):
```

Comment:
Classmethod here will give off issues when multiple python scripts would be using this code
There was levering on instances to avoid that from happening.
I'd suggest going back to that
This will give problems now:
```
vp_tester1 = VPoolTester()
vp_tester1.add_vpools()
vp_tester2 = VPoolTester()
vp_tester1.remove_vpools()  # This won't do anything as all it's references are cleared
```